Find a functionally equivalent code:android.appwidget.AppWidgetHost.createView:COMMENT
Method Modifier: public      final       
Comment:/**
 * Create the AppWidgetHostView for the given widget.
 * The AppWidgetHost retains a pointer to the newly-created View.
 */

@@ -1,7 +1,5 @@
 {
-    final int userId = mContext.getUserId();
     AppWidgetHostView view = onCreateView(mContext, appWidgetId, appWidget);
-    view.setUserId(userId);
     view.setOnClickHandler(mOnClickHandler);
     view.setAppWidget(appWidgetId, appWidget);
     synchronized (mViews) {
@@ -9,18 +7,13 @@
     }
     RemoteViews views;
     try {
-        views = sService.getAppWidgetViews(appWidgetId, userId);
-        if (views != null) {
-            views.setUser(new UserHandle(mContext.getUserId()));
-        }
+        views = sService.getAppWidgetViews(mContext.getOpPackageName(), appWidgetId);
     } catch (RemoteException e) {
         throw new RuntimeException("system server dead?", e);
     }

